packages/core/admin/admin/src/content-manager/pages/EditSettingsView/components/ModalForm.js

```diff
@@ -9,7 +11,11 @@ import { makeSelectModelAndComponentSchemas } from '../../App/selectors';
 import getTrad from '../../../utils/getTrad';
 import GenericInput from './GenericInput';
 
-const ModalForm = ({ onChange }) => {
+const FIELD_SIZES = [[4, '33%'], [6, '50%'], [8, '66%'], [12, '100%']];
```

Comment:
@iparidhi This will be displayed as percentages, which reflect the number of columns the field will be wide:

![Screenshot from 2022-02-10 12-32-04](https://user-images.githubusercontent.com/2244375/153400624-1d64cf7c-7d85-4e95-b231-d11a568a7716.png)

I'm wondering if you have a better idea here maybe?